spk/fuse-exfat/src/dsm-control.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# Package
+PACKAGE="exfat"
+DNAME="exfat"
```

Comment:
Please use consistent names. `exFAT` here, assuming that's what the project calls it.